release-please-config.json

```diff
@@ -12,11 +13,11 @@
     "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
-    "packages/sdk-codegen": { "release-as": "21.0.20" },
-    "packages/sdk-codegen-scripts": { "release-as": "21.0.21" },
-    "packages/sdk-codegen-utils": { "release-as": "21.0.13"},
+    "packages/sdk-codegen": { "release-as": "21.0.21" },
+    "packages/sdk-codegen-scripts": { "release-as": "21.0.22" },
+    "packages/sdk-codegen-utils": { "release-as": "21.0.12"},
     "packages/sdk-node": { },
-    "packages/sdk-rtl": { "release-as": "21.0.16" },
+    "packages/sdk-rtl": { "release-as": "21.0.17" },
```

Comment:
These changes are just to cover the next release version for `packages/sdk-*`. None of those packages should be affected by this PR so maybe it's distracting to have these changes in here but I thought I'd knock it out now that I had the file open

Reply:
Works for me